src/libaktualizr/uptane/managedsecondary.cc

```diff
@@ -27,6 +27,9 @@ bool ManagedSecondary::putMetadata(const MetaPack &meta_pack) {
   // No verification is currently performed, we can add verification in future for testing purposes
   detected_attack = "";
   current_meta = meta_pack;
+  if (current_meta.isConsistent()) {
```

Comment:
should be `if (!current_meta.isConsistent())`?

Reply:
Yes, thank You.